cmd/geth/les_test.go

```diff
@@ -66,7 +66,7 @@ func startGethWithRpc(t *testing.T, name string, args ...string) *gethrpc {
 	t.Logf("Starting %v with rpc: %v", name, args)
 	g.geth = runGeth(t, args...)
 	// wait before we can attach to it. TODO: probe for it properly
-	time.Sleep(1 * time.Second)
+	time.Sleep(2 * time.Second)
```

Comment:
What is happening here?

Reply:
"I also incremented a sleep in an unrelated test since the wait period for a geth instance to start was too low and it was making the test a bit flaky."

I could do it in a separate PR but honestly it was making me nervous

Reply:
I think since this change is totally unrelated to the rest of the PR, it would be better to not include it and instead we can submit a better fix (the probing) to upstream.

Reply:
fair, I'll remove it